core/src/test/java/io/questdb/test/cutlass/http/line/LineHttpSenderFailureTest.java

```diff
@@ -0,0 +1,139 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.test.cutlass.http.line;
+
+import io.questdb.client.Sender;
+import io.questdb.griffin.SqlException;
+import io.questdb.std.Misc;
+import io.questdb.std.Os;
+import io.questdb.std.str.Path;
+import io.questdb.test.AbstractBootstrapTest;
+import io.questdb.test.TestServerMain;
+import io.questdb.test.tools.TestUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.Closeable;
+import java.time.temporal.ChronoUnit;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class LineHttpSenderFailureTest extends AbstractBootstrapTest {
+
+    public static void scenarioRetryWithDeduplication(ServerController controller) throws Exception {
```

Comment:
nit: could be `private`

Reply:
it's used in EE tests

Reply:
Worth having a comment around that, at least.